test/unit/Reflection/Exception/NotAnObjectTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace BetterReflectionTest\Reflection\Exception;
+
+use BetterReflection\Reflection\Exception\NotAnObject;
+use BetterReflectionTest\ClassWithInterfaces;
+use BetterReflectionTest\ClassWithInterfacesOther;
+use PHPUnit_Framework_TestCase;
+
+/**
+ * @covers \BetterReflection\Reflection\Exception\NotAnObject
+ */
+class NotAnObjectTest extends PHPUnit_Framework_TestCase
```

Comment:
I added the test specifically to separate testing the message from the actual test that just verifies the `@throws`